From 3d3a90ec1deca80b9e4ae274a0a2f9aed2861df9 Mon Sep 17 00:00:00 2001
From: Quynh Nguyen <quynh.nguyen.xb@renesas.com>
Date: Tue, 14 Jul 2020 14:45:28 +0700
Subject: [PATCH 233/433] ARM: dts: r8a77470: Add SYSC PM domains for
 HSCIF{0|1|2}

Hook up hscif devices that are part of the CPG/MSTP Clock Domain to the
SYSC always-on PM Domain, for a more consistent device-power-area
description in DT.

Signed-off-by: Quynh Nguyen <quynh.nguyen.xb@renesas.com>
---
 arch/arm/boot/dts/r8a77470.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/r8a77470.dtsi b/arch/arm/boot/dts/r8a77470.dtsi
index b2d73e3..a104b8f 100644
--- a/arch/arm/boot/dts/r8a77470.dtsi
+++ b/arch/arm/boot/dts/r8a77470.dtsi
@@ -814,6 +814,7 @@
 			interrupts = <0 154 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&mstp7_clks R8A77470_CLK_HSCIF0>;
 			clock-names = "sci_ick";
+			power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
 			status = "disabled";
 		};
 
@@ -823,6 +824,7 @@
 			interrupts = <0 155 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&mstp7_clks R8A77470_CLK_HSCIF1>;
 			clock-names = "sci_ick";
+			power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
 			status = "disabled";
 		};
 
@@ -832,6 +834,7 @@
 			interrupts = <0 21 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&mstp7_clks R8A77470_CLK_HSCIF2>;
 			clock-names = "sci_ick";
+			power-domains = <&sysc R8A77470_PD_ALWAYS_ON>;
 			status = "disabled";
 		};
 
-- 
2.7.4

